DuckDuckGo/MainViewController.swift

```diff
@@ -164,6 +167,12 @@ class MainViewController: UIViewController {
         tabsButton.customView = tabSwitcherButton
     }
     
+    private func initBookmarksButton() {
+        gestureBookmarksButton.delegate = self
+        gestureBookmarksButton.image = UIImage(named: "Bookmarks")
+        bookmarksButton.customView = gestureBookmarksButton
```

Comment:
The one problem with current implementation is that Bookmarks button no longer has a "highlighted" state - there is no visual feedback in case it is pressed.

Reply:
added animation using image view alpha on touch..